decider-sample-fingerprinting/src/main/java/ca/on/oicr/pde/deciders/SampleFingerprintingDecider.java

```diff
@@ -50,7 +48,6 @@
     private String existingMatrix = "";
     private String templateTypeFilter = "";
     private String reseqTypeFilter = "";
-    private String manual_output = "false";
     private boolean separate_platforms = true;
     private boolean allow_singletons   = false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "allow_singletons" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)